### ISSUE TITLE
Linked SVGs don't have text - falling WCAG Success Criterion 2.4.4: Link Purpose (In Context)

### ISSUE THREAD

**Issue by AWearring** [localgovdrupal/localgov_publications#210]
The SVGs linked on a Publication Cover page fail[ WCAG Success Criterion 2.4.4: Link Purpose (In Context)](https://www.w3.org/WAI/WCAG20/Understanding/link-purpose-in-context) as they have a "non-empty accessible name" and fail [this ACT rule](https://www.w3.org/WAI/standards-guidelines/act/rules/c487ae/)

![image](https://github.com/user-attachments/assets/3fb89acd-9c6e-40f5-b62a-81ee0fea1741)
![image](https://github.com/user-attachments/assets/64723088-2117-482a-8559-d61819fdd7e4)

I propose we add a title attribute to the link

**Comment by AWearring** [comment]
Maybe like this: ` <a href="{{ svg_href }}" tabindex="-1" aria-hidden="true" title="{{ item.content['#title'] }}">`
@markconroy Any thoughts?

**Comment by markconroy** [comment]
I think we want assistive technology to ignore these links in the SVGs, that's why they are `aria-hidden`. Doing that, we then don't have 2 links (the icon and the page title) side-by-side going to the same place. That was from your issue a few weeks back - https://github.com/localgovdrupal/localgov_publications/issues/205

Presumably in your screenshot "Test page" is read out as the link text?

**Comment by AWearring** [comment]
The issue I'm highlighting is that the icons have no text content and therefore fail in that regard, however I'm not sure if this is an actual issue _if_ they are hidden. 
They are being highlighted as an issue by our Accessibility Audit tool (Monsido) and this would be solved by adding the title attribute, this would leave the links to be ignored by assistive technology due to the `aria-hidden` attribute. Of course there may be a better solution - such as combining both the icon and the current text into one link.

**Comment by markconroy** [comment]
I think we should add the `aria-hidden` attribute to the svg as well as the wrapping link tag. Those icons are decorative as far as I can see.

**Comment by AWearring** [comment]
I think we already have `aria-hidden` on the SVGs

**Comment by markconroy** [comment]
In the screenshot of the code you posted above, there is no aria-hidden on the SVG, only on the wrapping `<a>` tag.

**Comment by msayoung** [comment]
seems linked to #234

**Comment by msayoung** [comment]
Reviewing this in advance of Contrib day
I think it is now resolved on the demo site. 

it has aria-hidden and tabindex -1 on it. 
While I'd prefer it just wasn't a link, and then didn't this, this will do for now. 
Closing
